plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -25,6 +25,13 @@ library PaymentExitDataModel {
         mapping (uint192 => PaymentExitDataModel.StandardExit) exits;
     }
 
+    struct WithdrawData {
+        bytes32 outputGuard;
```

Comment:
This must be `exitTarget` or, something represents the address to withdraw.

Reply:
leave some discussion from slack:

rambo.pawel Today at 4:47 PM
@boolafish I think we can't store beneficiaries of exits when starting IFE, because that means we would have to provide output guard data to recover output owners from output guard, and that's private and may not be known by someone starting ife. We need to go with storing output gaurd, and then interested users provide output guard data on piggybacks @pepesza, is that correct? Maybe I am missing something about that privacy. (edited)
2 replies

rambo.pawel  < 1 minute ago
I talked to Andy, and output guard data is available for input transactions (already mined, included in chain), and may not be available for in-flight transaction itself. So we'll require to provide output guard data on piggyback.

rambo.pawel  < 1 minute ago
when starting IFE output guard data for input transactions is still needed for checking signatures (edited)

pepesza  1 minute ago
Yes, when owner is piggybacking, she needs to reveal the output guard data. This is a moment we fill in the blanks

pepesza  < 1 minute ago
Also, this mechanism only provides some privacy for funds at rest. When funds are being spend or are exited - data is being made public

---
Some notes agreement:
1. startIFE passes in the output guard pre-image. However, only used to check confirm sig (and would not be implemented in this PR). Beneficiaries is not checked at that time.
1. piggyback (input/output) checks the real beneficiaries of exit from output guard preimage and store it. Though input *can* do that while startIFE, move it to piggyback to make piggyback input/output logic more consistent.

Reply:
done

Reply:
sigh) I end up realize it is such a hassle to get output guard from unknown input tx. So I think probably we should only do such complex thing once. In my piggyback PR I end up asking for an extra array of outputGuards for input. 

like to discuss with you on this: https://github.com/omisego/plasma-contracts/pull/222/files#r318546401

Reply:
As we agree on having `exitTarget` set during inputs, let's sync on the data structure of our PRs so we can decrease merging conflicts. Below is my proposal:

```
    struct WithdrawData {
         bytes32 outputId;
         address payable exitTarget;
         address token;
         uint256 amount;
     }
```